emspay/webhook.php

```diff
@@ -89,6 +105,7 @@
         Db::getInstance()->update('emspay', array("id_order" => $id_order),
             '`ginger_order_id` = "'.Db::getInstance()->escape($ginger_order_id).'"');
 
-        $emspay->ginger->updateOrder($id_order,$order_details);
+        $order_details['merchant_customer_id'] = $id_order;
```

Comment:
Should there be a ['merchant_order_id']?

Reply:
fixed